plugins/gatsby-source-meetup/gatsby-node.js

```diff
@@ -0,0 +1,31 @@
+const crypto = require('crypto')
```

Comment:
Couldn't figure out the other Gatsby API plugins so now we have our own. Yay?

Reply:
Sure, why not? If it does the job 👍